website/docs/tutorials/deploying-java-app-with-database.md

```diff
@@ -64,13 +64,20 @@ In this example we will use odo to manage a sample [Java JPA MicroService applic
     cd ./application-stack-samples/jpa
     ```
 
-3. Initialize the application:
+3. In case of IBM Z & Power, CRW images are used in devfiles ([Please refer this](https://odo.dev/docs/tutorials/deploying-a-devfile-using-odo-on-IBM-Z-and-Power)), Hence for [devfile.yaml](https://github.com/OpenLiberty/application-stack-samples/blob/main/devfiles/maven-image/devfile.yaml), replace `maven:3.6-adoptopenjdk-11-openj9` with `registry.redhat.io/codeready-workspaces/plugin-java11-openj9-rhel8`:
```

Comment:
```suggestion
```suggestion
3. In case of IBM Z & Power, CRW images are used in devfiles, make sure to replace `maven:3.6-adoptopenjdk-11-openj9` image with `registry.redhat.io/codeready-workspaces/plugin-java11-openj9-rhel8` in [devfile.yaml](https://github.com/OpenLiberty/application-stack-samples/blob/main/devfiles/maven-image/devfile.yaml).
```

Reply:
I am confused here, if I am creating a component with `odo create java-openliberty` on IBM Z & Power, will I need to replace the image with the CRW image? If yes, then we can simply replace the image after `odo create java-openliberty sboproj`. What is the relevance of https://github.com/OpenLiberty/application-stack-samples/blob/main/devfiles/maven-image/devfile.yaml here?

Reply:
@valaparthvi Hi, because there is no public registry to provide the devfile.yaml for IBM  Z & Power, we only make sure and verified this devfile.yaml[https://github.com/OpenLiberty/application-stack-samples/blob/main/devfiles/maven-image/devfile.yaml] works fine with replace CRW image on IBM  Z & Power. Is this make sense?

Reply:
Okay, so will our steps be the following?
1. Fetch the devfile from https://github.com/OpenLiberty/application-stack-samples/blob/main/devfiles/maven-image/devfile.yaml
2. Replace the image in devfile with CRW image
2. odo create

Reply:
Yes. Sorry, there was some misunderstanding on my end, but I think things are clear to me now.